### ISSUE TITLE
Keyboard accessibility of user settings menu

### ISSUE THREAD

**Issue by lkiesow** [elan-ev/tobira#656]
The user settings menu in the top right corner has some accessibility problems for users needing a screen reader:

Using a keyboard to navigate in this menu, when I enter the menu, I would expect the first item of the opening sub-menu to be selected. Right now, it is kind of random what happens:
- Clicking the menu icon, the menu item stays focused instead of the first element in the opening menu
- Clicking on “Language” is worse
  - Using Talkback with Firefox or Chome on Android, I completely leave the page, starting again with the first accessible DOM element (logo) which is really annoying
  - Using Orca or just tab navigation in Firefox on Linux, I start with no obvious focus with the first language as the first element being focused after tabbing
  - Using Orca or just tab navigation in Chrome on Linux, I start with no obvious focus with “Back” as the first element being focused after tabbing

![Screenshot from 2023-01-12 22-25-50](https://user-images.githubusercontent.com/1008395/212186273-d605ff71-36ca-49a4-b7ce-e2163f8a5de4.png)

**Comment by owi92** [comment]
I think this is fixed now with the new design

**Comment by owi92** [comment]
Focus management should be fixed with #1001
